ut-user/index.js

```diff
@@ -16,12 +20,28 @@ function getCrypt(cryptKey) {
     return crypt;
 }
 
+function getIPAddress() {
+  var interfaces = os.networkInterfaces();
+  for (var devName in interfaces) {
+    var iface = interfaces[devName];
+
+    for (var i = 0; i < iface.length; i++) {
+      var alias = iface[i];
+      if (alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal)
+        return alias.address;
+    }
+  }
+  return '0.0.0.0';
+}
+
 module.exports = {
     init: function(b) {
         getCrypt(b.config.masterCryptKey);
         importMethod = b.importMethod.bind(b);
         checkMethod = b.config['identity.check'];
         debug = b.config.debug;
+        applicationVersion = b.config.applicationVersion;
```

Comment:
For the application version, we agreed to make use of the version of the implementation project. To make this application version available in the ut-identity module (or for that matter to be available to any module that constructs and manages the $meta object) we have to somehow extract this version from the package.json file and pass that value to the require module while that module is initializing. 

The one place that I can think of to extract this information and initialization is index.js file where we configure our ports and modules. Would it be fine if I modify the initialization line of ut-identity module to have a parameter to pass in and receive the application version at the module level. 

something similar to this,
![image](https://user-images.githubusercontent.com/24567041/27266586-b30a556a-54e5-11e7-8e62-eea63346a382.png)